packages/frontend/src/components/navbar/Navbar.tsx

```diff
@@ -57,97 +57,94 @@ export function Navbar(props: NavbarProps) {
         socialLinks={props.socialLinks}
       />
       {props.showBanner && <Banner />}
-      <nav
-        className={cx(
-          'relative flex h-14 items-center px-4 text-base lg:h-16 lg:justify-between lg:px-12',
-          'border-b border-gray-200 dark:border-gray-850',
-        )}
-      >
-        <button id="sidebar-menu-open" className="block lg:hidden">
-          <MenuOpenIcon className="block" aria-label="Open menu" />
-        </button>
-        <ul className="flex h-full items-center">
-          <li
-            className={cx(
-              props.showBridges
-                ? 'mx-4 lg:ml-0 lg:mr-8'
-                : 'absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2',
+      <nav className="lg:h-16border-b h-14 border-gray-200 text-base dark:border-gray-850">
+        <div className="relative mx-auto box-border flex h-full max-w-[1780px] items-center px-4 lg:justify-between lg:px-12">
```

Comment:
Nit pick: I would swap `div` and `nav` elements

Reply:
nice catch!